examples/configs/rm.yaml

```diff
@@ -0,0 +1,140 @@
+# Bradley-Terry (BT) Reward Model Training Configuration
+# (uses same base template as the SFT config but includes a new `reward_model_type` key that triggers Reward Model training)
+sft:
```

Comment:
Now that the `run_rm.py` script has been separated from `run_sft.py`, I would suggest to replace `sft:` with `rm:` here (also removing the comment above) and add something like `config.sft = "${.rm}"` in `run_rm.py` to make it compatible with the SFT functions.

(goal == make it more intuitive and future-proof, in case in the future we actually want to have different RM and SFT setings)